includes/Core/Util/Activation.php

```diff
@@ -199,27 +199,29 @@ private function get_activation_notice() {
 									method: 'POST',
 								} )
 									.then( function() {
-										$(self).prop( 'disabled', null );
+										event.target.disabled = null;
+										window.googlesitekitTrackingEnabled = !! checked;
 
 										var trackingId = googlesitekit.admin.trackingID;
-										var trackingScriptPresent = $( `script[src="https://www.googletagmanager.com/gtag/js?id=${ trackingId }"]` ).length > 0;
+										var trackingScriptPresent = document.querySelectorAll( `script[src="https://www.googletagmanager.com/gtag/js?id=${ trackingId }"]` ).length > 0;
 
 										if ( ! trackingScriptPresent ) {
-											$( 'body' ).append( `
+											document.body.insertAdjacentHTML( 'beforeend', `
 												\<script async src="https://www.googletagmanager.com/gtag/js?id=${ trackingId }"\>\</script\><?php // phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript ?>
+											` );
+											document.body.insertAdjacentHTML( 'beforeend', `
 												\<script\>
 													window.dataLayer = window.dataLayer || [];
 													function gtag(){dataLayer.push(arguments);}
 													gtag('js', new Date());
 													gtag('config', '${ trackingId }');
-													window.googlesitekitTrackingEnabled = true;
 												\</script\>
```

Comment:
Are these backslashes necessary? They aren't used in the similar code snippet above.

Reply:
If I didn't include them there was some weirdness with rendering. Might be because I'm using double-quotes and string interpolation here?

Reply:
Hmm 🤷‍♂